src/AdminConsole/Pages/Shared/Icons/Mail.cshtml

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns="http://www.w3.org/2000/svg" width="50" height="50" fill="none" viewBox="0 0 24 24">
```

Comment:
You can also dump the icon in `Components/Shared/Icons` with a `*.razor` file extension as a Razor component.

You'd then reference to it as follows:

```<component type="typeof(MailIcon)" render-mode="Static" />```

to add support for attributes you could add the `AdditionalAttributes` property to capture unmatched values: https://blazor-university.com/components/capturing-unexpected-parameters/

```<component type="typeof(MailIcon)" render-mode="Static" param-class="@("svg-class")" />```

But you'll always want to apply a style or class to it. If we then swap it out for a different SVG one day which could have a different width or height, at least it will look the same.

Reply:
@jonashendrickx wonderful, i'll do that. thank you!